cmd/kubernetes_create.go

```diff
@@ -175,6 +175,9 @@ var kubernetesCreateCmd = &cobra.Command{
 			for _, v := range strings.Split(removeapplications, ",") {
 				if utility.CheckAPPName(v) {
 					rmApp = append(rmApp, fmt.Sprintf("-%s", v))
```

Comment:
Sending `-` in applications field won't be a thing anymore in the API, so we should probably be removing this too.